### ISSUE TITLE
Cannot navigate the file list in side panel without mouse

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/jupyterlab-git#1186]
## Description

Navigating the list of staged/changed/etc files in side panel is not possible without mouse. The tab jumps to next section immediately.

## Reproduce

1. Open git side panel
2. Try to tab to a changed file to add it to staged
4. See focus jump over the list of files

## Expected behavior

Can use jupyterlab-git without mouse

## Context

- Python package version:

- Extension version:

- Git version:

- Operating System and its version:

<details><summary>Command Line Output</summary>
<pre>
Paste the output from your command line running `jupyter lab` here, use `--debug` if possible.
</pre>
</details>

<details><summary>Web Browser Output</summary>
<pre>
Paste the output from your browser web console here.
</pre>
</details>
